### PR TITLE
Updated SDK tools version to fix F5 deployment to experimental instance

### PR DESCRIPTION
**Pull request by RandomEngy**
Added .editorconfig to automatically enforce indent size of 2
Added .vs folder to .gitignore so it's not tracked
Updated targeted VS versions to include everything below 16
Did a little bit of cleanup in the .csproj

### CODE REVIEW

**Comment by FlorianDecker** [comment]
Thanks a lot for your pull requests! I will take a closer look at them tomorrow. Especially the fix for F5 Deployment was something which was bugging me for some time.

**Comment by RandomEngy** [comment]
The other PR includes the F5 fix, so you can ignore this one if you like.

File: LicenseHeaderManager/LicenseHeaderManager.csproj [hunk 1]
```diff
@@ -376,16 +370,16 @@
   <PropertyGroup>
     <UseCodebase>true</UseCodebase>
   </PropertyGroup>
-  <Import Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(MSBuildThisFileDirectory)..\packages\Microsoft.VisualStudio.Sdk.BuildTasks.14.0.14.0.23-pre\tools\VSSDK\Microsoft.VsSDK.targets" />
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != ''" />
```
Comment (FlorianDecker) [inline LicenseHeaderManager/LicenseHeaderManager.csproj]: Please import the "Microsoft.VsSDK.targets" from "..\packages\Microsoft.VSSDK.BuildTools.15.5.72\tools\VSSDK\Microsoft.VsSDK.targets" in the same fashion as Line 382. We want to have our VS specific dependencies in NuGet Packages so it is easier to build the Project on a BuildServer where VS might not be installed.

File: LicenseHeaderManager/VsixInformation/source.extension.vsixmanifest [hunk 2]
```diff
@@ -1,29 +1,29 @@
...
+        <License>VsixInformation/MIT.txt</License>
+        <GettingStartedGuide>https://github.com/rubicon-oss/LicenseHeaderManager/wiki</GettingStartedGuide>
+        <Icon>VsixInformation/Icon.png</Icon>
+        <PreviewImage>VsixInformation/Preview.png</PreviewImage>
+    </Metadata>
+    <Installation>
+        <InstallationTarget Id="Microsoft.VisualStudio.Community" Version="[12.0, 16.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Pro" Version="[11.0, 16.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Enterprise" Version="[14.0, 16.0)" />
+        <InstallationTarget Id="Microsoft.VisualStudio.Premium" Version="[11.0, 12.0]" />
```
Comment (FlorianDecker) [inline LicenseHeaderManager/VsixInformation/source.extension.vsixmanifest]: Why is it necessary to change this file? If i interpret https://blogs.msdn.microsoft.com/visualstudio/2013/08/08/update-for-extension-authors-vsix-manifest-version-range-attribute/ correctly, we seem to cover the same range of Versions (assuming that Minor Versions are not relevant, which they seem not to, as i have no problems installing or finding the License Header Manager extension on VS 15.4.0). 

Also please do remove the unnecessary indents

File: LicenseHeaderManager/LicenseHeaderManager.csproj [hunk 3]
```diff
@@ -1,31 +1,10 @@
...
-    <RunCodeAnalysis>true</RunCodeAnalysis>
-    <ErrorReport>prompt</ErrorReport>
-    <CodeAnalysisRuleSet>MinimumRecommendedRules.ruleset</CodeAnalysisRuleSet>
-    <DeployExtension>False</DeployExtension>
-  </PropertyGroup>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" ToolsVersion="14.0">
+  <Import Project="..\packages\Microsoft.VSSDK.BuildTools.15.5.72\build\Microsoft.VSSDK.BuildTools.props" Condition="Exists('..\packages\Microsoft.VSSDK.BuildTools.15.5.72\build\Microsoft.VSSDK.BuildTools.props')" />
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
   <PropertyGroup>
+	<VsixType>v3</VsixType>
```
Comment (FlorianDecker) [inline LicenseHeaderManager/LicenseHeaderManager.csproj]: Please fix indentation

File: LicenseHeaderManager/LicenseHeaderManager.csproj [hunk 4]
```diff
@@ -35,16 +14,24 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>LicenseHeaderManager</RootNamespace>
     <AssemblyName>LicenseHeaderManager</AssemblyName>
+	<Prefer32Bit>false</Prefer32Bit>
```
Comment (FlorianDecker) [inline LicenseHeaderManager/LicenseHeaderManager.csproj]: Please fix indentation

File: LicenseHeaderManager/LicenseHeaderManager.csproj [hunk 5]
```diff
@@ -35,16 +14,24 @@
...
     <TargetFrameworkVersion>v4.0</TargetFrameworkVersion>
+    <GeneratePkgDefFile>true</GeneratePkgDefFile>
     <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
     <FileUpgradeFlags>
     </FileUpgradeFlags>
     <UpgradeBackupLocation>
     </UpgradeBackupLocation>
     <OldToolsVersion>12.0</OldToolsVersion>
-    <TargetFrameworkProfile />
+	<TargetFrameworkProfile />
```
Comment (FlorianDecker) [inline LicenseHeaderManager/LicenseHeaderManager.csproj]: Please fix indentation

**Comment by FlorianDecker** [comment]
We prefer one Pull Request per issue, for history purposes, so 2 Pull Requests are fine.

File: LicenseHeaderManager/VsixInformation/source.extension.vsixmanifest (hunk 2 above)
Comment (RandomEngy) [reply LicenseHeaderManager/VsixInformation/source.extension.vsixmanifest]: In that file it says: "] – maximum version inclusive." That means to me that it includes 15.0 but nothing beyond that. Now they may have special logic to allow minor numbers beyond the configured maximum of 15.0, but "16.0)" more clearly states the intent. But I can change it back if you want.

As for the indents, I just used the Visual Studio designer to save the file. I think it makes sense to leave the indent the way the tool writes it. I have my XML tab settings set to Insert Spaces and size 2, so I think the tool is just insisting on that. But again, I can revert the whole file if you insist.

**Comment by RandomEngy** [comment]
Okay. I can rebase the second one after this goes in.

File: LicenseHeaderManager/VsixInformation/source.extension.vsixmanifest (hunk 2 above)
Comment (RandomEngy) [reply LicenseHeaderManager/VsixInformation/source.extension.vsixmanifest]: Changed the rest of them as well.

**Comment by FlorianDecker** [comment]
Merged! Thank you very much for your contribution.

Please rebase the other pull request on top of this so we can review the changes seperately. I will comment on the other pull request shortly.

**Pull request merged**